packages/contentstack-import/src/import/modules/extensions.ts

```diff
@@ -83,7 +83,12 @@ export default class ImportExtensions extends BaseClass {
       return;
     }
 
-    const apiContent = values(this.extensions);
+    const apiContent = [];
+
+    for (const [uid, content] of Object.entries(this.extensions)) {
+      (content as Record<string, any>).uid = uid;
+      apiContent.push(content);
+    }
```

Comment:
Hey @antonyagustine , This code we don't require here.  While rewriting the extension export we are storing the data in different format .